File: server/Models/VaultKeep.cs
namespace keeprFINAL.Models;

public class VaultKeep : RepoItem<int>
{
	// NOTE: RepoItem fills in the Id
	// public string Id { get; set; }

	public string CreatorId { get; set; }
	public int KeepId { get; set; }
	public int VaultId { get; set; }
}

File: client/src/models/VaultKeep.js
import { DatabaseItem } from "./DatabaseItem.js";

export class VaultKeep extends DatabaseItem {
  constructor(data) {
    super(data) //calls constructor on DataBaseItem

    // id, createdAt, updatedAt inherited from DatabaseItem

		this.creatorId = data.creatorId
		this.keepId = data.keepId
		this.vaultId = data.vaultId
  }
}
